packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardRenderer.js

```diff
@@ -434,6 +450,9 @@ const AdaptiveCardRenderer = ({ actionPerformedClassName, adaptiveCard, disabled
   }, [adaptiveCard]);
 
   useEffect(() => {
+    setAttribute(adaptiveCardElementRef.current, 'aria-pressed', 'true') &&
+      setAttribute(adaptiveCardElementRef.current, 'role', 'button');
```

Comment:
If the `aria-pressed` already has value of `true`, it won't set `role` to `button`. Is this intentional?

Reply:
fixed